config/samples/deployment.yaml

```diff
@@ -511,6 +511,7 @@ rules:
   - create
   - update
   - delete
+  - patch
```

Comment:
This file is generated through kustomize, so I think this change will just get overridden later. Can you add it to the rbac_role.yaml files and run `make` to add them to all the generated files? There are a few different files with the `manager-role` role, and I think we want to add it to all of them.

Reply:
Sure! :smile: